src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20171002_cluster-init-bucket-column.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="hussein" id="cluster-init-bucket">
+        <addColumn tableName="CLUSTER">
+            <column name="initBucket" type="varchar(1024)"/>
```

Comment:
Maybe make this not-null? A cluster must have an init bucket, right?

Reply:
Also, Unique?